printer_mods/Minsekt/Rear_Umbilical/README.md

```diff
@@ -0,0 +1,14 @@
+# Rear Umbilical Addon
+
+| BOM | Pieces |
+| ------ | ------ |
+| M3 Heatinsert | 4x |
+| M3x10 SCHS | 4x |
+
+![umbilical](https://github.com/Minsekt/moronvods/blob/main/Rear_Umbilical/images/rear_umbilical.png)
```

Comment:
Please make this a relative link. `./images/rear_umbilical.png`

Reply:
fixed